### DISCUSSION TITLE
Need help hacker

### DISCUSSION THREAD

**Discussion by Bearroller9** [community/community#142735] [Discussions] [answered]
### Select Topic Area

Question

### Body

I don't know anything about this site but ex friend has put on Trojans last Nov and I can't get rid of him he's using this site plus Linux Netscape Firefox and other to hack everything he's been in my mortgage and last month uploaded hidden child documents. The police have been contacted 11 times but won't help cam Anyone help me please. I don't know Anything about coding.

**Comment by WhiteWolfDD** [comment]
Download the **System Informer** [Here](https://www.systeminformer.com/) to monitor all active processes, identify any suspicious ones, and terminate them as needed.

Alternatively, you can perform a full system reinstallation with disk reformatting for a clean start.

If you have any questions, don't hesitate to ask. I'm here to help.

**Reply by Bearroller9** [reply]
Downloaded it to the phone not sure what to do now

**Reply by WhiteWolfDD** [reply]
> Downloaded it to the phone not sure what to do now

Did you download a trojan to your phone, or an analog of System Informer?

If a trojan is installed on your phone, you need to reset your phone to factory settings. This may not help if your phone is jailbroken. If your phone is jailbroken, reinstall the official firmware for your phone model.

If you downloaded an analog of System Informer, check the process list and delete any .apk files that you suspect. Alternatively, an easier option is to simply reset your phone to factory settings if you believe there’s a trojan on your phone.

**Comment by evilgensec** [comment]


**Reply by Bearroller9** [reply]
Hacker put malware which contained at least 15 Trojans with gps tracking and krysyrok8ng can't rid Pps and sites of them

**Comment by ihateapples** [comment]
cooked

**Comment by MasterInQuestion** [comment]
Tossing your device in fire might be more sensible than above instructions...

    ----

    Air-gap works.
    If your concern is just leak of sensitive information.

**Comment by samus-aran** [comment, marked as answer]
Hi @Bearroller9 , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Reply by MasterInQuestion** [reply]
It feels like a general issue about being hacked, however.
    Nothing GitHub specific.

**Comment by KhairulHasanRakib** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> I don't know anything about this site but ex friend has put on Trojans last Nov and I can't get rid of him he's using this site plus Linux Netscape Firefox and other to hack everything he's been in my mortgage and last month uploaded hidden child documents. The police have been contacted 11 times but won't help cam Anyone help me please. I don't know Anything about coding.

When we install any software, it's preset possible this type of path:

C:\Users\GIGABYTE\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\
C:\ProgramData\Microsoft\Windows\Start Menu\Programs\
C:\Program Files

Remove unnecessary things.

Click the Windows button, type system configuration, and you will see the services, startup, and tools. If anything looks unnecessary, uncheck the theme.

again type remote desktop connection and show options and the advanced tab server authentication set does not connect. (if this remote connection you don't need)

**Comment by Bearroller9** [comment]
No the problem is I have no clue how tid site works and there is tons of
codes from this site on all my phone and no way of knowing what or who the
user is can someone email and givee step by sep
